### PR TITLE
Simplify AdditionalSharedFrameworkToInstallArguments in workloads-testing.targets

### PR DESCRIPTION
**Pull request by akoeplinger**
We can use the same args everwhere and also remove an outdated comment.

### CODE REVIEW

File: eng/testing/workloads-testing.targets [hunk 1]
```diff
@@ -50,11 +50,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!--<AdditionalSharedFrameworkToInstallArguments Include="-Version 8.0.0-rc.2.23457.7" />-->
     <!-- Required for running apps built with 9.0 sdk, but the sdk does
          not yet support *running* with 9.0 sdk -->
```
Comment (am11) [inline eng/testing/workloads-testing.targets:54]: Is this comment about `dotnet run` or 'current SDK running with .NET 9 SDK'? I can't seem to reproduce this locally. .NET 9 SDK publish the app and I can run it from publish folder as well as with `dotnet run`.

Comment (akoeplinger) [reply eng/testing/workloads-testing.targets:54]: Looks like it was done to fix this issue back then: https://github.com/dotnet/runtime/pull/93066#issuecomment-1749449349

@radical do we still need this?

**Review by ViktorHofer — APPROVED** [review]
Approving but as @am11 mentioned, we might not need this code anymore.

File: eng/testing/workloads-testing.targets (hunk 1 above)
Comment (radical) [reply eng/testing/workloads-testing.targets:54]: Yes we need this but for different reasons - to build+run 8.0 apps with 8.0 bits. But the comment can be removed.

**Pull request merged**